Workbooks/Perimeter81OverviewWorkbook.json

```diff
@@ -0,0 +1,175 @@
+{
```

Comment:
I tested and reviewed the workbook. Overall there are two visuals hence looks pretty lean. Here are a few ideas. Would be great to have 6 or more visuals that can enable customers to work with Perimeter81 data effectively. 
1. Trend charts - activity patterns over time - you can have multiple trends to show different activity types trending over time. 
2. Including IPs in the pivot based on different types of activities. 
3. Patterns with application types and activity types - you can include user info in that mix as well
4. I see pivot by email - you can add a pivot by user name (first and last name) with a different visual type
5. Extension of trends to include user and activity patterns over time.
6. If you have in product visuals on Perimeter 81, it'll be great to include those as well.

Reply:
Added 4 more charts (latest commit)